Adyen/Service/Payment.cs

```diff
@@ -97,5 +98,12 @@ public async Task<PaymentResult> Authorise3DS2Async(PaymentRequestThreeDS2 payme
             var jsonResponse = await _authorise3DS2.RequestAsync(jsonRequest, requestOptions);
             return JsonConvert.DeserializeObject<PaymentResult>(jsonResponse);
         }
+
+        public AuthenticationResultResponse AuthenticationResult(AuthenticationResultRequest authenticationResultRequest)
+        {
+            var jsonRequest = JsonConvert.SerializeObject(authenticationResultRequest);
+            var jsonResponse = _getAuthenticationResult.Request(jsonRequest);

```

Comment:
I would expect GetAuthenticationResultResponse and GetAuthenticationResultRequest here since the endpoint is /getAuthenticationResult, but that's how it was autogenerated, right?

Reply:
The method name can be GetAuthenticationResult. The models AuthenticationResultResponse and AuthenticationResultRequest are generated from open api specs with this name

Reply:
The method was called just AuthenticationResult in the beginning, but docs team changed it to GetAuthenticationResult afterwards... the models remained the same though :(